lambdanotes/src/App.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import './App.css';
+import { Route } from 'react-router-dom';
+
+import notesList from './notes-list.js';
+import Sidebar from './components/Sidebar/Sidebar.js';
+import ListView from './components/ListView/ListView.js';
+import CreateNote from './components/CreateNote/CreateNote.js';
+import ViewNote from './components/ViewNote/ViewNote.js';
+import EditNote from './components/EditNote/EditNote.js';
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      notesList: [],
+      currentID: 0
```

Comment:
I assume you're using this to keep track of which single note you're looking at? Very nice :+1: